src/config.js

```diff
@@ -203,6 +203,41 @@ export const BENCHMARKS = {
     labels: ['Firefox (tiering)', 'Firefox (wasm-baseline)', 'Firefox (wasm-ion)', 'Chrome'],
     label: 'WebAssembly Embenchen',
   },
+  'wasm-godot': {
+    compare: {
+      'raptor-wasm-godot-firefox': {
+        color: '#e55525',
+        label: 'Firefox (tiering)',
+        frameworkId: RAPTOR_FRAMEWORK_ID,
+        suite: 'raptor-wasm-godot-firefox',
+        buildType: 'opt',
+      },
+      'raptor-wasm-godot-baseline-firefox': {
+        color: 'red',
+        label: 'Firefox (wasm-baseline)',
+        frameworkId: RAPTOR_FRAMEWORK_ID,
+        suite: 'raptor-wasm-godot-baseline-firefox',
```

Comment:
Hi @bnjbvr, Would you mind telling where you got information against Godot benchmark? I'm working on another fix, just saw this PR and I am curious about this. And how this dashboard knows about `raptor-wasm-godot-firefox` or `raptor-wasm-godot-baseline-firefox` or any benchmark we add?

Excuse me for this naive question but I'm still exploring the code and trying to understand while solving different issues.

Reply:
Sure! It's because we added the benchmark as part of [bug 1493648](https://bugzilla.mozilla.org/show_bug.cgi?id=1493648), which defines the jobs (thus the constant strings you're seeing here). Then I mostly cargo-culted what other benchmarks looked like, and it appeared to work fine.

Reply:
If you have other questions, @armenzg will probably have more context than I do.

Reply:
Thanks, @bnjbvr :) It helped. I will ping Armen for queries.